algorithms/schema-free/closeness_cent_approx_query.gsql

```diff
@@ -0,0 +1,238 @@
+CREATE QUERY closeness_approx (
+    SET<STRING> v_type, 
+    SET<STRING> e_type,
+		int k,  # sample num
```

Comment:
Please include default parameter values whenever you can, e.g. debug=0.
For the core algorithms, use your judgment. When you really don't want to set a limit, I typically set the default to a negative value, and add logic in the algorithm to handle that.

Reply:
Use the same defaults as those in closeness_cent, if you can. Also, change the order of the parameters to be like the other schema free algorithms. Parameter order tries to follow:
1. Input
2. Algorithm body
3. Output